README.md

```diff
@@ -6,11 +6,22 @@ The goal is to study the features required for such identification, when working
 the code of this project is arrange according to the architecture shown in https://github.com/asilev/NLP_FinalProject/blob/master/doc/architecture/NLP%20Author%20Classification%20Architecture.png.
 
 The files in TwitterReader are used to retrieve data sets from Twitter, using the twitter4j framework.
-This folder is meant to be compiled and run as a standalone executable with its own main method.
+The Twitter4j framework provides a pre-compiled JAR library, twitter4j-core-4.0.4.jar, which is imported into the project and then allows the access and extraction of data.
```

Comment:
According to the submission instruction, we are supposed to write in the readme file how to run our deliverables, and you wrote here how we used the framework.
I think it is important to mention which parts are meant to be run as a stand alone part (have their own main), and relate to the parameters our main function requires.
It is less important to explain how we used the framework, as this will not help understand how to run our parts.